kalite/distributed/static/js/distributed/rating/views.js

```diff
@@ -146,11 +146,24 @@ var StarView = BaseView.extend({
     template: require("./hbtemplates/star.handlebars"),
 
     events: {
-        "click .star-rating-option": "rate_value_callback"
+        "click .star-rating-option": "rate_value_callback",
+        "click .star-rating-option >": "rate_value_callback",
+        "mouseenter .star-rating-option >": "mouse_enter_callback",
+        "mouseenter .star-rating-option": "mouse_enter_callback",
+        "mouseleave .star-rating-option": "mouse_leave_callback"
+    },
+
+    label_values: {
+        1: "Very Low",
+        2: "Low",
+        3: "Normal",
+        4: "High",
+        5: "Very High"

```

Comment:
@mcgallaspy did you see the comments on models.py in the previous, merged
PR?
On Aug 18, 2015 3:55 PM, "Michael Gallaspy" <notifications@github.com>
wrote:

> In kalite/distributed/static/js/distributed/rating/views.js
> <https://github.com/learningequality/ka-lite/pull/4248#discussion_r37363735>
> :
>
> > @@ -146,11 +146,24 @@ var StarView = BaseView.extend({
> >      template: require("./hbtemplates/star.handlebars"),
> >
> >      events: {
> > -        "click .star-rating-option": "rate_value_callback"
> > +        "click .star-rating-option": "rate_value_callback",
> > +        "click .star-rating-option >": "rate_value_callback",
> > +        "mouseenter .star-rating-option >": "mouse_enter_callback",
> > +        "mouseenter .star-rating-option": "mouse_enter_callback",
> > +        "mouseleave .star-rating-option": "mouse_leave_callback"
> > +    },
> > +
> > +    label_values: {
> > +        1: "Very Low",
> > +        2: "Low",
> > +        3: "Normal",
> > +        4: "High",
> > +        5: "Very High"
>
> Star rating labels are set here. This could easily be configurable on a
> per-view basis.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/learningequality/ka-lite/pull/4248/files#r37363735>.
>